queries/futures/utils.ts

```diff
@@ -238,11 +240,13 @@ export const getReasonFromCode = (reasonCode?: BigNumber): MarketClosureReason |
 		case 3:
 		case 55:
 		case 65:
+		case 231:
 			return 'circuit-breaker';
 		case 99999:
 			return 'emergency';
 		default:
-			return null;
+			console.log(Number(reasonCode));
```

Comment:
Shall we remove this `console.log` before merging?

Reply:
done